README.md

```diff
@@ -2,42 +2,160 @@
+By default, the output file is created in the same folder as the input file, and will have the same name as the input file but with a different extension. 
+TTX will _never_ overwrite existing files, but if necessary will append a unique number to the output filename (before the extension) such as `Arial#1.ttf`
+
+When using TTX from the command line there are a bunch of extra options, these are explained in the help text, as displayed when typing `ttx -h` at the command prompt. 
+These additional options include:
+
+* specifying the folder where the output files are created
+* specifying which tables to dump or which tables to exclude
+* merging partial `.ttx` files with existing `.ttf` or `.otf` files
+* listing brief table info isntead of dumping to `.ttx`
+* splitting tables to separate `.ttx` files
+* disabling TrueType instruction disassembly
+
+### The TTX file format
+
+The following tables are currently supported:
+
+    BASE, CBDT, CBLC, CFF, COLR, CPAL, DSIG, EBDT, EBLC, FFTM, GDEF, 
+    GMAP, GPKG, GPOS, GSUB, JSTF, LTSH, MATH, META, OS/2, SING, SVG, 
+    TSI0, TSI1, TSI2, TSI3, TSI5, TSIB, TSID, TSIJ, TSIP, TSIS, TSIV, 
+    VDMX, VORG, avar, cmap, cvt, feat, fpgm, fvar, gasp, glyf, gvar, 
+    hdmx, head, hhea, hmtx, kern, loca, ltag, maxp, meta, name, post, 
+    prep, sbix, vhea, vmtx
+
+Other tables are dumped as hexadecimal data.
+
+TrueType fonts use glyph indices (GlyphIDs) to refer to glyphs in most places.
+While this is fine in binary form, it is really hard to work with for humans. 
+Therefore we use names instead.
+
+The glyph names are either extracted from the `CFF ` table or the `post` table, or are derived from a Unicode `cmap` table. 
+In the latter case the Adobe Glyph List is used to calculate names based on Unicode values. 
+If all of these methods fail, names are invented based on GlyphID (eg `glyph00142`)
+
+It is possible that different glyphs use the same name. 
+If this happens, we force the names to be unique by appending `#n` to the name (`n` being an integer number.)
+The original names are being kept, so this has no influence on a "round tripped" font.
+
+Because the order in which glyphs are stored inside the binary font is important, we maintain an ordered list of glyph names in the font.
+
+### Development and feedback
+
+TTX/FontTools development is ongoing, with an active community of developers including professional developers employed at major software corporations and type foundries as well as hobbyists. 
+
+Feature requests and bug reports are always welcome at <https://github.com/behdad/fonttools/issues/>
+
+The best place for discussions about TTX from an end-user perspective as well as TTX/FontTools development is the <https://groups.google.com/d/forum/fonttools> mailing list.
+You can also email Behdad privately at <behdad@behdad.org>
```

Comment:
Its all over the web already; https://www.google.com/search?q=behdad%40behdad.org